doc/tutorial/faq_tutorial.txt

```diff
@@ -64,7 +64,7 @@ Defining the theano function
 
 .. code-block:: python
 
-   f = theano.function(..., updates=updates)
```

Comment:
Ok, it's just really confusing because of the name of the variables and the text below.  It seem to imply that we can have a lookup table on updates to make partial updates or something of the sort, which is apparently not the case.